### PR TITLE
Holiest of all PR's...

### PR DESCRIPTION
**Pull request by Fortifier42**
The FortyFriar

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java [hunk 1]
```diff
@@ -125,5 +125,6 @@ public void onPlayerItemTakesDamage(PlayerItemDamageEvent event) {
         fire();
         event.setCancelled(cancelled);
         event.setDamage(damage.asInt());
+        event.getPlayer().updateInventory();
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java]: I'll ask again... has this been tested for /both/ cancellation cases?

IE:

- Does it work right /when cancelled/?
- Dos it work right /when NOT cancelled/?

Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java]: What could the issue be? Cause I've tested it both times, the item still takes damage if the event isn't cancelled. if it is, it doesn't take damage..

Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java]: That's all I was asking :)

Comment (Morphan1) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java]: To clarify, have you checked with vast amounts and visually confirmed? As in, changing the damage to 50% of the durability, but also cancelling it to make 100% sure etc

Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java]: Yes. Both visually and textually with tags.

**Pull request merged**